scripts/source-maintenance.sh

```diff
@@ -15,39 +15,55 @@
 ## Source Code Format Enforcement
 #
 # A checker to recursively reformat all source files: .h .c .cc .cci
-# using a custom astyle formatter and to use MD5 to validate that
-# the formatter has not altered the code syntax.
+# using a custom astyle formatter and to use md5sum (or similar) to validate
+# that the formatter has not altered the code syntax.
 #
 # If code alteration takes place the process is halted for manual intervention.
 #
 
-# whether to continue execution after a failure
 # TODO: Expand the subset of failures covered by this feature; see run_().
 KeepGoing="no"
 # the actual name of the directive that enabled keep-going mode
 KeepGoingDirective=""
-#
-# The script checks that the version of astyle is TargetAstyleVersion.
-# if it isn't, the default behaviour is to not perform the formatting stage
-# in order to avoid unexpected massive changes if the behaviour of astyle
-# has changed in different releases.
-# if --with-astyle /path/to/astyle is used, the check is still performed
-# and a warning is printed, but the sources are reformatted
+
+# the version of astyle tool required by Squid coding style
 TargetAstyleVersion="3.1"
-ASTYLE='astyle'
 
 # whether to check and, if necessary, update boilerplate copyright years
 CheckAndUpdateCopyright=yes
 
 printUsage () {
-    echo "Usage: $0 [option...]"
-    echo "options:"
-    echo "    --keep-going|-k"
-    echo "    --check-and-update-copyright <yes|no>"
-    echo "    --with-astyle </path/to/astyle/executable>"
+cat <<_EOF
+Usage: $0 [option...]
+options:
+    --keep-going|-k                            (default: stop on error)
+    --check-and-update-copyright <yes|no>      (default: no)
+    --with-astyle </path/to/astyle/executable> (default: astyle)
```

Comment:
Documentation does not quite match the code:

```suggestion
    --with-astyle </path/to/astyle/executable> (default: astyle-${TargetAstyleVersion} or astyle)
```